src/media/css/lib/mixins.styl

```diff
@@ -186,8 +186,25 @@ category-icon-position($slug, $scale=1) {
     background-position: ($positions[0] * $cat-icon-size-default * $scale * -1) ($positions[1] * $cat-icon-size-default * $scale * -1);
 }
 
+ltr() {
```

Comment:
I tend to prefer "assume LTR until RTL overrides" but wouldn't enforce it on anyone.